assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -41,10 +41,6 @@ che.workspace.no_proxy=
 # Configures dns resolving servers. May contain several values separated by comma sign.
 che.dns.resolvers=
```

Comment:
I think It's a good time to prefix everything related to docker infra with `che.infra.docker`. @garagatyi WDYT? Shall we create an issue for that?

Reply:
I would love to. But what about backward compatibility? I think we've got something for that recently.

Reply:
Aliases mechanism can provide backward compatibility? Here is an PR where it was done https://github.com/eclipse/che/pull/8636